lib/plugins/aws/lib/monitorStack.js

```diff
@@ -25,127 +24,131 @@ module.exports = {
 
     this.serverless.cli.log(`Checking Stack ${action} progress...`);
 
-    return new BbPromise((resolve, reject) => {
-      async.whilst(
-        () => validStatuses.indexOf(stackStatus) === -1,
-        callback => {
-          const cfStackStatusCheckFrequency =
-            options.frequency || process.env.SLS_AWS_MONITORING_FREQUENCY || 5000;
+    return new Promise((resolve, reject) => {
+      const recursivePromiseFn = () => {
+        if (validStatuses.indexOf(stackStatus) === -1) {
```

Comment:
Let's use `includes` instead (although I know originally it was `indexOf`, still it's a bad pattern after we have `includes` on board)